spec/reek/context/code_context_spec.rb

```diff
@@ -90,18 +90,16 @@
         end
       end
 
-      context 'with no block' do
-        it 'returns an empty array of ifs' do
-          expect(ctx.each_node(:if, [])).to be_empty
-        end
+      it 'returns an empty array of ifs when no block is passed' do
```

Comment:
Nice. Overall we had / have way to many 1-example contexts, something we kill throughout our spec base.